test/backend_graph_comparison.in.cpp

```diff
@@ -45,98 +45,178 @@ class serialized_graph_files : public ::testing::TestWithParam<string>
 public:
     void compare_results(NodeVector& result_nodes,
                          vector<shared_ptr<runtime::Tensor>> ref_results,
-                         vector<shared_ptr<runtime::Tensor>> bk_results)
+                         vector<shared_ptr<runtime::Tensor>> bk_results,
+                         vector<shared_ptr<runtime::Tensor>> bk_isolated_results)
     {
         for (int i = 0; i < ref_results.size(); ++i)
         {
             const shared_ptr<runtime::Tensor>& ref_data = ref_results.at(i);
             const shared_ptr<runtime::Tensor>& bk_data = bk_results.at(i);
+            const shared_ptr<runtime::Tensor>& bk_isolated_data = bk_isolated_results.at(i);
 
-            cout << "Comparing results for " << result_nodes.at(i)->get_name() << endl;
-            if (auto node = dynamic_pointer_cast<op::GetOutputElement>(result_nodes.at(i)))
+            std::shared_ptr<ngraph::Node> result_node = result_nodes.at(i);
+            cout << "Comparing results for " << result_node->get_name() << endl;
+            if (result_node->get_arguments().size() > 0)
             {
-                cout << "  Parent node: ";
-                for (auto& p : node->get_arguments())
+                cout << "  inputs:" << endl;
+                for (auto& p : result_node->get_arguments())
                 {
-                    cout << " " << p->get_name() << endl;
-                    cout << "   nargs: " << p->get_arguments().size() << endl;
+                    cout << "    " << p->get_name() << " " << p->get_element_type() << endl;
                 }
             }
 
+            // Sanity check testing code
```

Comment:
Are these still desired? if so, change or remove the comment

Reply:
Comment tweaked for better clarity.